index.rst

```diff
@@ -609,7 +609,13 @@ Display Components
     ST7735, components/display/st7735, st7735.jpg
     ST7789V, components/display/st7789v, st7789v.jpg
     ST7920, components/display/st7920, st7920.jpg
-    ILI9xxx, components/display/ili9xxx, ili9341.jpg
```

Comment:
I dont think we should remove this entry as it is still something people might look for once actually using the code

Reply:
👍🏻